tox.ini

```diff
@@ -36,6 +41,11 @@ deps =
     isort==5.10.1
 commands = isort --check-only --diff .
 
+[testenv:jsonformat]
+deps =
+    pytest==6.2.5
```

Comment:
Should have spotted this in a previous PR but why not using the requirements file? Less to install?

Reply:
Yes, the idea is to install as little as possible in each environment and let requirements.txt contain the runtime dependencies. That's how I prefer to do things but I'm not religious about it.